content/en/docs/components/katib/hyperparameter.md

```diff
@@ -412,7 +413,7 @@ Follow the steps as described for the _random algorithm example_
 
 ## Cleanup
 
-To delete Katib from Kubernetes cluster run::
+To delete Katib from Kubernetes cluster run:
 
```

Comment:
@andreyvelich Hi!

On Lines 424-430 - please double check your hyperlinks. Also, can we refactor this for accessibility/inclusivity ("see" -> "refer to" or "go to" or similar)?

For example:

```
- Learn to configure and run your experiment with Katib in the
  [running an experiment](/docs/components/hyperparameter-tuning/experiment/) guide.
- About the [Katib configuration file (Katib config)](/docs/components/hyperparameter-tuning/katib-config/).
- How to [set up environment variables for each Katib component](/docs/components/hyperparameter-tuning/env-variables/).
```

@andreyvelich Please double check your links. I think `/docs/components/hyperparameter-tuning/...` work but the older `/docs/components/katib/`? don't (not any more, at least). Thanks!

Reply:
Update: The links are changing to `docs/components/kativ` - Got it!